### ISSUE TITLE
Add the "sort" command to lits/sets/sorted-sets

### ISSUE THREAD

**Issue by twidi** [limpyd/redis-limpyd#83]
The only way to use the powerful "sort" command of redis is to use something like that:

``` python
myobj.database.connection.sort(myobj.myfield.key, **other_paras)
```

We could add the "sort" command to the lists/sets/sorted-sets to help doing simple sort (simple = sorting a list with alpha=True, for example... not simple = using the other sort feature, as we need keys)

So to sort a list we could do (here an alphanumeric sort for the example):

```
myobj.myfield.sort(alpha=True)
```

**Comment by twidi** [comment]
Fixed on develop

**Comment by twidi** [comment]
Published in 1.1
